Assets/Scripts/GlobalValues/WordStore.cs

```diff
@@ -5,10 +5,17 @@ static class WordStore
 {
     private static List<string> WORDS;
     private static readonly TextAsset WORDTEXTFILE = Resources.Load<TextAsset>("WordStore/WordList");
+    private static readonly TextAsset FRENCHWORDTEXTFILE = Resources.Load<TextAsset>("WordStore/FrenchWordList");
 
-    public static void GenerateWords()
+    public static void GenerateWords(string language)
     {
-        WORDS = new List<string>(WORDTEXTFILE.text.Replace("\r\n", "\n").Split('\n'));
+        if(language == "English")
```

Comment:
If we change the word list name to something like "WordList_English.txt" and "WordList_French.txt" we could probably do something like 
```
private static string PATH_PREFIX = "WordStore/WordList_"
private static string FILE_EXT = ".txt"

public static void GenerateWords(string language) {
    var filePath = PATH_PREFIX + language + FILE_EXT;
    TextAsset WordList = Resources.Load<TextAsset>(filePath);
    WORDS = new List<string>(WordList .text.Replace("\r\n", "\n").Split('\n'));
}
```

Obviously with some extra checks for if the language is null or if the file does not exist to load in a default file instead or something like that.

With this all we have to do is drop in a wordlist file then add the language in options and it should be loaded in, with the weights missing

Reply:
Yeah think i tried to explain this, this way we would just have to make sure we save the list in a consistent way, i can impliment this